app/Http/Kernel.php

```diff
@@ -48,6 +48,10 @@ class Kernel extends HttpKernel
             // TODO \Illuminate\Routing\Middleware\ThrottleRequests::class.':web',
         ],
 
+        'inertia' => [
+            Middleware\HandleInertiaRequests::class,
+        ],
```

Comment:
It might be enough to add a `middlewareAliases` below if we do not plan on adding more to this group. I don't think it's necessary to create a group.

Reply:
Good point, the `middlewareAliases` stuff below totally escaped me. Addressed in https://github.com/RetroAchievements/RAWeb/pull/2502/commits/0323a0b522942f22d464b0d4e9bf5e3175778c64.